excise/utils.py

```diff
@@ -585,17 +596,43 @@ def process_checkout_metadata(
     metadata_key = get_metadata_key("checkout_metadata_")
     data_cache_key = f"{metadata_key}{checkout.token}"
 
-    metadata = {
-        get_metadata_key("itemized_taxes"): json.dumps(
-            taxes_data.get("TransactionTaxes")
-        ),
-        get_metadata_key("tax_transaction"): json.dumps(
-            taxes_data.get("Transaction")
-        ),
-    }
+    metadata = build_metadata(taxes_data)
 
     if force_refresh or metadata_requires_update(metadata, data_cache_key):
         checkout.refresh_from_db()
         checkout.store_value_in_metadata(items=metadata)
         checkout.save()
         cache.set(data_cache_key, metadata, cache_time)
+
+
+def get_sales_tax(taxes_data: Dict[str, Dict]) -> int:
+    """
+    Build the sales tax amount where tax type is either "S" or "U"
+    """
+    return sum(
+        map(
+            lambda tax_data: tax_data.get("TaxAmount", 0),
+            filter(
+                lambda tax_data: tax_data.get("TaxType") == "S"
+                or tax_data.get("TaxType") == "U",
+                taxes_data.get("TransactionTaxes", []),
+            ),
+        )
+    )
+
+
+def get_other_tax(taxes_data: Dict[str, Dict]) -> int:
+    """
+    Building the remaining tax amount after getting the sales tax,
+    i.e. where the tax type is neither "S" nor "U"
+    """
+    return sum(
+        map(
+            lambda tax_data: tax_data.get("TaxAmount", 0),
+            filter(
+                lambda tax_data: tax_data.get("TaxType") != "S"
```

Comment:
Shouldn't we use const values for `U` and `S`? 
Also, would be good to have a meaningful const name or description which will describe what `U` and `S` means.

Reply:
Could you please review the latest changes [here](https://github.com/saleor/saleor-plugin-avatax-excise/pull/12/commits/03a711ff1080efd544e2cee4dd47a7994e30388e)?

Reply:
Looks great!